.github/scripts/build.sh

```diff
@@ -64,3 +64,5 @@ echo =====
 ${DOCKER} bash -c "cd build && cpack -G TGZ -D CPACK_PACKAGE_FILE_NAME=psidk-ubuntu-${UBUNTU_BUILD_VER}"
 echo =====
 ${DOCKER} bash -c "cd build && mv book psidk-book && tar czf ../psidk-book.tar.gz psidk-book"
+echo =====
+${DOCKER} bash -c "cargo generate -p ./package-templates/ --destination ./services/user/ --init -v --name Buildtest --silent basic-01 && cd services/user/Buildtest && cargo-psibase package"
```

Comment:
Is `cargo-psibase` in the `PATH`?

Reply:
Only in `psibase-contributor`.
Do you care if it's added to path in the builder image? Or just use full path?

Reply:
Yes. Good check.
<img width="633" alt="image" src="https://github.com/user-attachments/assets/ec40151e-a298-4a8c-8838-e1711c168523" />

Reply:
I misread this. I've added the absolute path.